draft-irtf-cfrg-hpke.md

```diff
@@ -605,6 +607,19 @@ def Context.Open(aad, ct):
   return pt
 ~~~~~
 
+## Secret Export {#kpke-export}
+
+HPKE provides a interface for exporting secrets from the encryption Context, similar
+to the TLS 1.3 exporter interface (See {{8446}}, Section 7.5). This interface takes as
+input a context string `exporter_context` and desired length `L` (in octets), and produces
+a secret derived from the internal exporter secret using the corresponding KDF Expand
+function.
+
+~~~~~
+def Context.Export(exporter_context, L):
+  return Expand(self.exporter_secret, exporter_context, L)
```

Comment:
Note that this is different from the TLS exporter, which has both a label and a context.  Might not need all of that here, though.

```
   TLS-Exporter(label, context_value, key_length) =
       HKDF-Expand-Label(Derive-Secret(Secret, label, ""),
                         "exporter", Hash(context_value), key_length)
```